subsys/net/lib/openthread/platform/radio.c

```diff
@@ -50,6 +56,38 @@ static struct ieee802154_radio_api *radio_api;
 static s8_t tx_power;
 static u16_t channel;
 
+enum net_verdict ieee802154_radio_handle_ack(struct net_if *iface,
+					     struct net_pkt *pkt)
+{
+	ARG_UNUSED(iface);
+
+	size_t ack_len = net_pkt_get_len(pkt);
+
+	if (ack_len != ACK_PKT_LENGTH) {
+		return NET_CONTINUE;
```

Comment:
Log something?

Reply:
It could lead to a log flood on some devices. While it could work with nRF driver, which calls this function explicitly for ACK frames, there are drivers that call `ieee802154_radio_handle_ack` for every single frame, and decide whether it was an ACK frame based on the result. For instance [here](https://github.com/zephyrproject-rtos/zephyr/blob/f2cb20c772b3700896cf0f67b7e307677866058e/drivers/ieee802154/ieee802154_uart_pipe.c) (and it's not a sole example).